dmgroups.tex

```diff
@@ -135,29 +135,47 @@ \subsection{DM Leadership Team \label{sect:dmlt}}
+The Technical Baseline, including software/hardware and documentation, is produced by DM and controlled by the DMCCB.
 DMCCB validates that the form and content of the Technical Baseline is consistent with LSST project standards such as the Systems Engineering Management Plan (SEMP) \citeds{LSE-17}.
 
 \begin{itemize}
-\item Charter/purpose
-	\begin{itemize}
-	\item Ensure that the DM Technical Baseline (LDM-xxx) documents are baselined and subsequently changed only when necessary and according to LSST and DM configuration control processes
-	\end{itemize}
-\item Membership
+\item Responsibilities:
 	\begin{itemize}
-	\item Chaired by the Systems Engineer (\secref{role:sysengineer}).
-	\item The DMCCB has the same membership as the Systems Engineering Team (\secref{sect:sysengt}).
-	\item For on-line virtual meetings, if a consensus or quorum or is not reached within one week, the DM Project Manager will make a unilateral decision
+	\item Determine when deliverables (controlled documents and software) are ready to be baselined (placed under configuration controlled status) or released. This include LDM series documents.
+	\item Review and approve/reject proposed changes to baselined items
+        \item Review and approves all RFCs prior to 'Adoption'.
+	\item Monitor and approve  DM software releases
+        \item Monitor DM Jira issues status.
+	\item Ensure that the DM  follows  LSST and DM configuration control processes.
 	\end{itemize}
-\item Responsibilities
+\item Membership:
 	\begin{itemize}
-	\item Determines when specification and deliverables are of sufficient maturity and quality to be baselined (placed under configuration controlled status) or released.
-	\item Reviews and approves/rejects proposed changes to baselined items
+        \item Core members:
+                \begin{itemize}
+                \item DM Project Manager
+                \item DM Subsystem Scientist
+	        \item Chaired by the Systems Engineer (\secref{role:sysengineer}).
+                \item Operations Architect
+                \item Software Architect
+                \item DM Configuration and Release Manager (secretary)
+                \end{itemize}
+        \item Optional members are involved when topics to discuss are relevant to their areas of expertise:
+                \begin{itemize}
+                \item DMPM Deputy, when DMPM is not available
+                \item DM Deputy Subsystem Scientist, when DM Subsystem Scientist is not available
+                \item Pipeline Scientist
+                \item Science Pipelines / Interface Scientist
+                \item T/CAMs, who can delegate to their deputies
+                \end{itemize}
+	\item For on-line virtual meetings, if a consensus or quorum is not reached within one week, the DM Project Manager will make a unilateral decision
+        \item DMPM can also make unilateral decisions in cases of urgency. In that case DMCCB will assess the change a \textit{posteriori}.
```

Comment:
“a posteriori” is all Latin, so if you're going to italicise the “posteriori” probably do the “a” too!

What does this actually mean? Does the DMCCB have the right to reverse decisions made by the DMPM if its “a posteriori” analysis doesn't like them?

Reply:
DMCCB need to be informed and check that the change has no impacts on others DM components.